From 219666315cc93eda4a2139ee1d5de4650a9b6a59 Mon Sep 17 00:00:00 2001
From: Jin Wei <wei.a.jin@intel.com>
Date: Thu, 21 Mar 2013 14:12:15 +0800
Subject: [PDK MR2 Temp] Enable GUI for PDK

BZ: 117199

We have to lunch the surfaceflinger in init.rc not by system_init

Change-Id: Ifd12f3c41bbef65c1e5084a64d550c91e25b0eca
Signed-off-by: Jin Wei <wei.a.jin@intel.com>
---
 init.rc |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/init.rc b/init.rc
index 29b7b0f..d86085c 100644
--- a/init.rc
+++ b/init.rc
@@ -438,11 +438,11 @@ service debuggerd /system/bin/debuggerd
 #    user root
 #    group radio cache inet misc audio log
 
-#service surfaceflinger /system/bin/surfaceflinger
-#   class main
-#   user system
+service surfaceflinger /system/bin/surfaceflinger
+   class main
+   user system
 #   group graphics drmrpc
-#   onrestart restart zygote
+   onrestart restart zygote
 
 service zygote /system/bin/app_process -Xzygote /system/bin --zygote --start-system-server
     class main
-- 
1.7.4.1

